### DISCUSSION TITLE
Budibase and PostgreSql at DigitalOcean

### DISCUSSION THREAD

**Discussion by adriano168** [Budibase/budibase#5512] [Help]
I installed Budibase and Postgresql on 'cloud.digitalocean', I can't make the connection and fetch the tables, it returns the error: ''Error updating datasource schema''

**Comment by melohagan** [comment]
Hey @adriano168 

Would you be able to provide any errors from the Network tab? (Fn+F12 on chrome)

It could also be helpful to see the schema of the PostgreSQL table.

**Reply by adriano168** [reply]
I couldn't solve it, can you help me? The Postgresql database and Budibase, both are hosted on the Digital Ocean server, but I couldn't make the connection! If necessary I grant access to my account, I would like to learn! however few people know about budibase!

**Reply by hendrikjap** [reply]
Did you enable SSL?

**Reply by adriano168** [reply]
> Você ativou o SSL?

if it is the same as in the image, I tried both ways

what I can not understand the reason? when I use the connection with budibase hosted on DigitalOcean, it doesn't connect to the bank!

![Erro3](https://user-images.githubusercontent.com/100057246/164760438-1500628e-f24f-49e4-aec9-33c100cc1435.png)
![Erro4](https://user-images.githubusercontent.com/100057246/164760468-ee1a62f4-2c9a-4dde-b0b7-4a5ad4a2b0cb.png)

**Reply by jonnymccullagh** [reply]
@adriano168 you haven't mentioned whether you have postgres running on an additional Digital Ocean server or if it is the same server that is running Budibase. If it is the same server then you could try `127.0.0.1` as the Host IP address. 
If it is on another server then try temporarily switching off any firewalls. You could also test connectivity by SSHing into the Budibase server and try using telnet to reach the database port e.g.
```
telnet 167.212.30.2 5432
```
This would help to determine if the connectivity is problematic regardless of Budibase. Please post your output.
